beacon-chain/core/altair/sync_committee.go

```diff
@@ -212,10 +212,10 @@ func ValidateSyncMessageTime(slot types.Slot, genesisTime time.Time, clockDispar
 	// Verify sync message slot is within the time range.
 	if messageTime.Before(lowerBound) || messageTime.After(upperBound) {
 		return fmt.Errorf(
-			"sync message slot %d not within allowable range of %d to %d (current slot)",
+			"sync message slot %d not within allowable range of slots %d to %d",
```

Comment:
I think the current slot part was helpful, What do you think?
```suggestion
			"sync message slot %d not within allowable range of slots %d to %d (current slot)",
```